src/types/bancor.d.ts

```diff
@@ -764,6 +768,9 @@ export interface ViewProtectedLiquidity {
   coverageDecPercent: number;
   fullCoverage: number;
   givenVBnt?: string;
+  pendingReserveReward: BigNumber;
```

Comment:
Can we get away by returning this as a ```string```? 
Just to match consistency with all the other values passed + if we were to ever leave ```BigNumber``` for another lib it would probably be less of a pain in the ass.

Reply:
sure, I'll change it.